bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -459,7 +459,12 @@ <h2 class="tween">
         <div class="intro tween">
           {% if l10n_has_tag('mwc_2015') %}
             <h4>{{ _('A superior mobile experience') }}</h4>
-            <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+            <div class="play-button"> 
+              <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+              <small class="download-other">
+                <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+              </small>
```

Comment:
I have found a winning combination for this that I will propose in a new bug for discussion.